packages/react-liveness/src/components/FaceLivenessDetector/shared/Hint.tsx

```diff
@@ -69,7 +69,7 @@ export const Hint: React.FC<HintProps> = ({ hintDisplayText }) => {
     [FaceMatchState.TOO_MANY]: hintDisplayText.hintTooManyFacesText,
     [FaceMatchState.TOO_CLOSE]: hintDisplayText.hintTooCloseText,
     [FaceMatchState.TOO_FAR]: hintDisplayText.hintTooFarText,
-    [FaceMatchState.MATCHED]: undefined,
+    [FaceMatchState.MATCHED]: hintDisplayText.hintHoldFaceForFreshnessText,
```

Comment:
Why has this changed?

Reply:
Previously when in the `MATCHED` state we did not show any hint text or progress bar which was the source of the flickering because you need to stay in the matched state for a period of time. If you were to move out of the `MATCHED`state the progress bar would re-appear with the hint text and thus the source of the flickering when moving between states